references.go

```diff
@@ -114,6 +108,19 @@ func walkGraph(result *[]*object.Commit, seen *map[plumbing.Hash]struct{}, curre
 	return nil
 }
 
+func derivedFromOneParent(path string, current *object.Commit, parents []*object.Commit) (bool, error) {
+	for _, parent := range parents {
+		different, err := differentContents(path, current, []*object.Commit{parent})
```

Comment:
`differentContents` already iterates the list? Why do you need to pass each of the parents individually? Can't you just pass `parents`?